pkg/logging/field.go

```diff
@@ -3,15 +3,15 @@ package logging
 import "go.uber.org/zap"
 
 type DataField interface {
-	getField() zap.Field
+	GetField() zap.Field
```

Comment:
This was causing a compiler fail because ./package/logging/logctx does not have access to the private interface method.

Reply:
where is this being used / causing compilation error?

Reply:
Running `go test ./...` was failing on this line: https://github.com/caring/go-packages/blob/9f21459a24379f4f15d1c59e37f5c5c877426e9d/pkg/logging/logctx/context.go#L30

Reply:
I figured out the issue was because `TagsToFields` in `context.go` it needed to use `logging.DataField` instead of `logging.Field`. i have reverted `getField` to private.